### ISSUE TITLE
(3.0.62) Error drag/drop of method

### ISSUE THREAD

**Issue by dalehenrich** [ericwinger/Jade#443]
This is a method that was originally drag/dropped into this class and the error is occurring when I try to drag the method back to the original class:
![image](https://user-images.githubusercontent.com/623951/53039563-49a4ed80-3434-11e9-8e5c-1904ed7b32ff.png)

Here's the server stack:
```
a ArgumentError occurred (error 2283), reason:rtErrInvalidArgClass, nil is not one of the class kinds in anArray( String, DoubleByteString).
ArgumentError (AbstractException) >> _signalWith: @5 line 25
ArgumentError (AbstractException) >> signal @2 line 47
UndefinedObject (Object) >> _error:args: @15 line 11
UndefinedObject (Object) >> _validateClasses: @10 line 9
UndefinedObject class (Behavior) >> _primCompileMethod:symbolList:category:oldLitVars:intoMethodDict:intoCategories:intoPragmas:environmentId: @2 line 53
UndefinedObject class (Behavior) >> _primitiveCompileMethod:symbolList:category:oldLitVars:intoMethodDict:intoCategories:intoPragmas:environmentId: @15 line 25
UndefinedObject class (Behavior) >> compileMethod:dictionaries:category:intoMethodDict:intoCategories:intoPragmas:environmentId: @2 line 34
[] in  ExecBlock0 (RwGsPlatform) >> _parseMethod:category:using:environmentId: @4 line 11
ExecBlock0 (ExecBlock) >> onSynchronous:do: @2 line 14
RwGsPlatform >> _parseMethod:category:using:environmentId: @9 line 18
[] in  ExecBlock0 (RwGsPlatform) >> parseSelectorFrom: @4 line 6
ExecBlock0 (ExecBlock) >> on:do: @3 line 42
RwGsPlatform >> parseSelectorFrom: @2 line 17
RwMethodDefinition class >> newForSource:protocol: @3 line 4
RwPrjBrowserTool >> addOrUpdateMethod:inProtocol:forClassNamed:isMeta:inPackageNamed: @4 line 13
RwPrjBrowserTool >> addOrUpdateMethod:inProtocol:forClassNamed:isMeta: @13 line 37
GsuGsDevKit_3_5_0_Upgrade class (Behavior) >> rwCompileMethod:category: @9 line 4
[] in  ExecBlock0 (RowanClassService) >> compileMethod:behavior:symbolList:inCategory: @2 line 6
ExecBlock0 (ExecBlock) >> on:do: @3 line 42
[] in  ExecBlock0 (RowanClassService) >> compileMethod:behavior:symbolList:inCategory: @2 line 7
ExecBlock0 (ExecBlock) >> on:do: @3 line 42
[] in  ExecBlock0 (RowanClassService) >> compileMethod:behavior:symbolList:inCategory: @2 line 9
ExecBlock0 (ExecBlock) >> on:do: @3 line 42
RowanClassService >> compileMethod:behavior:symbolList:inCategory: @2 line 11
RowanClassService >> saveMethodSource:category: @17 line 11
RowanClassService (RowanService) >> servicePerform:withArguments: @2 line 2
RowanClassService >> servicePerform:withArguments: @4 line 4
[] in  ExecBlock1 (JadeServer) >> updateFromSton: @10 line 12
Array (Collection) >> do: @5 line 10
JadeServer64bit32 (JadeServer) >> updateFromSton: @7 line 6
GsNMethod class >> _gsReturnToC @1 line 1
```
The error is occuring because the `methodString` is `nil` ...  it seems that the nil is coming from the client:
![image](https://user-images.githubusercontent.com/623951/53039927-144ccf80-3435-11e9-8a1b-ae0c25b71534.png)

Marking Feb, for review ...

**Comment by dalehenrich** [comment]
It seems that if I do the drag/drop operation without first selecting the method, I get the error ...

**Comment by ericwinger** [comment]
I can reproduce it. It was recently introduced with the recent optimizations. Will be fixed in the next release. Workaround is to select the method first before dragging (as you note).

**Comment by ericwinger** [comment]
Fixed in 3.0.63 https://github.com/GemTalk/Jadeite/commit/cda254d5906f19d66f33e903b617f2935419
